xbmc/music/windows/GUIWindowVisualisation.cpp

```diff
@@ -118,6 +126,61 @@ bool CGUIWindowVisualisation::OnAction(const CAction &action)
       return true;
     }
     break;*/
+
+    case REMOTE_0:
+    case REMOTE_1:
+    case REMOTE_2:
+    case REMOTE_3:
+    case REMOTE_4:
+    case REMOTE_5:
+    case REMOTE_6:
+    case REMOTE_7:
+    case REMOTE_8:
+    case REMOTE_9:
+      {
```

Comment:
haven't had a chance to verify, but i suspect this is c+p from the other window and that it needs to be extracted so we don't end up with dupe code.